payments/voucher_manager.go

```diff
@@ -0,0 +1,136 @@
+package payments
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// paymentStatus stores the status of payments for a given payment channel.
+type paymentStatus struct {
+	channelSender   common.Address
+	channelReceiver common.Address
+	startingBalance *big.Int
+	largestVoucher  Voucher
+	currentBalance  Balance
+}
+
+// VoucherManager receives and generates vouchers. It is responsible for storing vouchers.
+type VoucherManager struct {
+	channels map[types.Destination]*paymentStatus
+	me       common.Address
+}
+
+// NewVoucherManager creates a new voucher manager
+func NewVoucherManager(me types.Address) *VoucherManager {
+	channels := make(map[types.Destination]*paymentStatus)
+	return &VoucherManager{channels, me}
+}
+
+// Register registers a channel for use, given the sender and starting balance of the channel
```

Comment:
This comment needs updating to reflect the inclusion of the receiver in the arguments.

Reply:
Done in https://github.com/statechannels/go-nitro/pull/844/commits/659ea22d1d8624973eb4e08cc0f41515b2744c78